lib/core/test/unit/Cardano/Wallet/Api/Malformed.hs

```diff
@@ -1487,6 +1512,9 @@ instance Malformed (BodyParam (ApiBytesT 'Base64 SerialisedTx))
 -- no cases here as all bad requests are served by ErrDecodeSignedTxWrongPayload
 -- in Server.hs. Tested by integration tests.
 
+instance Malformed (BodyParam (ApiT SealedTx)) where
+    malformed = [] -- fixme: add a couple things
```

Comment:
Can this be filled in at this stage?

Reply:
I think this might be more difficult that it seems, only an OctetStream is accepted by the endpoint. I'm not sure BodyParam has been designed around this use-case. Investigating further.

Reply:
The default request body for this machinery is to use JSON as the content type:

```
defaultApiRequest :: Request
defaultApiRequest = defaultRequest
    { requestHeaders =
        [ (hContentType, "application/json")
        , (hAccept, "*/*")
        ]
    }
```

I think I need to modify:

```
    type MkBodyRequest (ReqBody '[ct] a :> sub) = BodyParam a -> IO (MkBodyRequest sub)
    gEveryBodyParam _ req b =
        gEveryBodyParam (Proxy @sub) <$> (setRequestBody b req)
```

To pattern match on `OctetStream` on the type-level and set the content-type accordingly.

Not sure on how to do that yet.

Reply:
This seems to be working, so glad I did some research on Servant a little while ago:

```
instance {-# OVERLAPPING #-}
    ( GEveryEndpoints sub
    ) => GEveryEndpoints (ReqBody '[OctetStream] a :> sub)
  where
    gEveryEndpoint _ =
        gEveryEndpoint (Proxy @sub)

    type MkPathRequest (ReqBody '[OctetStream] a :> sub) = MkPathRequest sub
    gEveryPathParam _ =
        gEveryPathParam (Proxy @sub)

    type MkBodyRequest (ReqBody '[OctetStream] a :> sub) = BodyParam a -> IO (MkBodyRequest sub)
    gEveryBodyParam _ req b =
        gEveryBodyParam (Proxy @sub) <$> (setRequestBody b req {
            requestHeaders = [ (hContentType, "application/octet-stream")
                             , (hAccept, "*/*")
                             ]
        })

    type MkHeaderRequest (ReqBody '[OctetStream] a :> sub) = Header "Content-Type" OctetStream -> MkHeaderRequest sub
    gEveryHeader _ req (Header h) =
        gEveryHeader (Proxy @sub) $ req
            { requestHeaders = requestHeaders req ++ [(hContentType, h)] }

```

Reply:
Great, so if we put a few examples like this:
```haskell
malformed = [("", _todo1), ("hello", _todo2), ("cafecafe", _todo3)]
```
then the tests pass??